### PR TITLE
[Bug Fix] This is to fix an indexing issue causing problems in qc7/8

### PR DESCRIPTION
**Pull request by bregnery**
This is a problem observed in qc7/8 but not in my own venv.

## Description
This should fix the sbitThresh analysis

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context
Required to work in the qc7/8 environment

## How Has This Been Tested?
```bash
#!/usr/bin/env bash

#=========================================================================================
# Testing Suite for the Inflection Point finder ------------------------------------------
#=========================================================================================


echo "Welcome to Brendan's Testing Suite for the Inflection Point Finder"
echo "------------------------------------------------------------------------------------"
echo "This script executes a bunch of files known to cause issues for the inflection"
echo "point finder"
echo "------------------------------------------------------------------------------------"

echo "Did you remember to update??? With make pip and pip install rpm*.tar.gz?"

echo "File 1"
echo "This tests how the code handles a vfat with a bad vfat"
echo "Please make sure (2,5,4) is filled in your system_specfic_constants.py file"

# This file causes issues because of a zero slope issue
anaSBitThresh.py /data/bigdisk/GEM-Data-Taking/GE11_QC8//sbitRate/channelOR/2019.10.25.13.59/SBitRateData.root

echo "File 2"
echo "This tests how the code handles a detector with missing vfats"
echo "Please make sure (1,2,0) is filled in your system_specfic_constants.py file"

# This file is from the coffin where the detector is missing vfats
anaSBitThresh.py /data/bigdisk/GEM-Data-Taking/GE11_Integration//sbitRate/channelOR//2019.10.24.17.45/SBitRateData.root
```

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jsturdy** [comment]
What are the versions of dependencies in the two environments?

**Comment by bregnery** [comment]
how do I figure that out?

When I re-based, I noticed new dependencies.

The old version accepted the following syntax `dict_dacInflectPts[dacName][ohKey][vfat][0]` which returned `[15.0]`, which I see is not quite right, but it handled it. Then some dependency was added that would not accept this input and I had to change it to `dict_dacInflectPts[dacName][ohKey][vfat][0][0]` so it returned `15.0` which is what I really meant for it to be in the first place.

**Comment by jsturdy** [comment]
probably something like `pip freeze` from your environment, as well as the same for QC8
mostly interested in `numpy`/`root_numpy`/`pandas`, not sure there are other relevant deps

**Comment by bregnery** [comment]
This bug also does not affect the p5 computer

**Comment by bregnery** [comment]
For numpy:
```bash
(inflectionEnv)-bash-4.2$ pip show numpy
Name: numpy
Version: 1.14.6
```

**Comment by jsturdy** [comment]
* which version is at P5?
* which version(s) is(are) on QC8/7 in the environment(s) that is(are) being used there?
* which package's API changed and in which version?

once these are answered, the appropriate changes to the requirements must be made as well

**Comment by jsturdy** [comment]
OK, I'm going to make an executive decision and we'll run `numpy==1.16.5` and `root_numpy==4.8.0` on our production machines (so code **must** be tested to work with at least these)

**Comment by jsturdy** [comment]
@bregnery needs another rebase, and can you make sure it works as expected based on the aforementioned versions?

**Comment by bregnery** [comment]
Okay, it works with the new requirements. Now I just need to re-base.

**Comment by bregnery** [comment]
Done

**Review by jsturdy — COMMENTED** [review]
Can you update the relevant requirements files? (`requirements.txt`, `pkg/setup.cfg`, etc)

**Comment by jsturdy** [comment]
@bregnery bump

**Comment by bregnery** [comment]
updated and it compiles/runs. But @jsturdy, I think that the requirements might not be getting parsed correctly.

File: pkg/setup.cfg [hunk 1]
```diff
@@ -71,7 +71,7 @@ requires = python >= 2.6
            numpy >= 1.7
```
Comment (jsturdy) [inline pkg/setup.cfg]: this version should match the one in the requirements, which i think can be `>=1.14` for the features actually in use

File: pkg/setup.cfg [hunk 2]
```diff
@@ -71,7 +71,7 @@ requires = python >= 2.6
...
            python-pandas
            python-sqlalchemy
-           root_numpy >= 4.7
+           root_numpy >= 4.8
```
Comment (jsturdy) [inline pkg/setup.cfg]: this also doesn't need to change, things work fine with `4.7`

**Review by jsturdy — CHANGES_REQUESTED** [review]

**Review by jsturdy — APPROVED** [review]
:+1:

**Pull request merged**